var/spack/repos/builtin/packages/hermes-shm/package.py

```diff
@@ -39,7 +39,8 @@ class HermesShm(CMakePackage):
     # Required deps
     depends_on("catch2@3.0.1")
     depends_on("yaml-cpp")
-    depends_on("doxygen@1.9.3", type="build")
+    depends_on("doxygen@1.9.3:", type="build")
```

Comment:
Tested: The current versions of `doxygen` are compatible with this older versions.
Build also succeeds with `doxygen@1.11.0`, hence it does not need to be pinned to a single version.